wsd/QuarantineUtil.cpp

```diff
@@ -78,6 +78,48 @@ namespace Quarantine
         return size;
     }
 
+    void deleteExpiredFiles(std::vector<std::string> &files)
+    {
+        if (!isQuarantineEnabled())
+            return;
+
+        // parameter is used in this function to avoid creating and sorting file list multiple times
+        if(files.empty())
```

Comment:
We have a single call site surely ? and by the time we get here it has a list of all the files surely (?)